### PR TITLE
Update header in Ukrainian translation

### PR DESCRIPTION
**Pull request by sakhnik**
Ukrainian part in the pull request #3040 
Update the translation too.

### CODE REVIEW

File: src/po/uk.cp1251.po [hunk 1]
```diff
@@ -12,15 +12,15 @@
...
-"POT-Creation-Date: 2017-07-12 12:22+0300\n"
+"POT-Creation-Date: 2018-06-20 06:30+0300\n"
 "PO-Revision-Date: 2010-06-18 21:53+0300\n"
 "Last-Translator: ������� ����� <sakhnik@gmail.com>\n"
-"Language-Team: Bohdan Vlasyuk <bohdan@vstu.edu.ua>\n"
+"Language-Team: Ukrainian\n"
 "Language: uk\n"
 "MIME-Version: 1.0\n"
-"Content-Type: text/plain; charset=cp1251\n"
+"Content-Type: text/plain; charset=UTF-8\n"
```
Comment (k-takata) [inline src/po/uk.cp1251.po]: Is this change correct?
The filename is `uk.cp1251.po`.

Comment (sakhnik) [reply src/po/uk.cp1251.po]: Thanks for pointing out!

**Comment by brammool** [comment]
> Ukrainian part in the pull request #3040 

Thanks.  Did you contact Bohdan?

-- 
I AM THANKFUL...
...for the clothes that fit a little too snug because it
means I have more than enough to eat.

 /// Bram Moolenaar -- Bram@Moolenaar.net -- http://www.Moolenaar.net   \\\
///        sponsor Vim, vote for features -- http://www.Vim.org/sponsor/ \\\
\\\  an exciting new programming language -- http://www.Zimbu.org        ///
 \\\            help me help AIDS victims -- http://ICCF-Holland.org    ///

**Comment by sakhnik** [comment]
@brammool, actually, never. I remember trying back ten years ago, but unsuccessfully.

**Comment by brammool** [comment]
> @brammool, actually, never. I remember trying back ten years ago, but unsuccessfully.

I CC'ed him, hopefully he replies.

Please run the check script, I found one error: the Plural-Forms header
is missing.  And it looks like the translations use 2 plural forms,
while the recommended one for Ukrainian has three forms.
Please check that.


-- 
A fool learns from his mistakes, a wise man from someone else's.

 /// Bram Moolenaar -- Bram@Moolenaar.net -- http://www.Moolenaar.net   \\\
///        sponsor Vim, vote for features -- http://www.Vim.org/sponsor/ \\\
\\\  an exciting new programming language -- http://www.Zimbu.org        ///
 \\\            help me help AIDS victims -- http://ICCF-Holland.org    ///

**Comment by sakhnik** [comment]
@brammool, thanks! And thus I learn my native language with Vim.

**Comment by chrisbra** [comment]
I think this has been included as of https://github.com/vim/vim/commit/675e8d6adb88d5b38087870dfda51c0205070741, so closing.

**Comment by brammool** [comment]
Thanks for contacting me, Bram! I guess I need to update my email address,
how can I do that? Also, how can I help? :)

**Pull request closed without merging**